worlds/sc2/Options.py

```diff
@@ -873,16 +870,9 @@ def get_excluded_missions(world: World) -> Set[SC2Mission]:
     excluded_missions: Set[SC2Mission] = set([lookup_name_to_mission[name] for name in excluded_mission_names])
 
     # Excluding Very Hard missions depending on options
-    if (get_option_value(world, "exclude_very_hard_missions") == ExcludeVeryHardMissions.option_true
-        ) or (
-            get_option_value(world, "exclude_very_hard_missions") == ExcludeVeryHardMissions.option_default
-            and (
-                    mission_order_type not in [MissionOrder.option_vanilla_shuffled, MissionOrder.option_grid]
-                    or (
-                            mission_order_type == MissionOrder.option_grid
-                            and get_option_value(world, "maximum_campaign_size") < 20
-                    )
-            )
+    if (world.options.exclude_very_hard_missions == ExcludeVeryHardMissions.option_true) or (
+            world.options.exclude_very_hard_missions == ExcludeVeryHardMissions.option_default
+            and mission_order_type in dynamic_mission_orders and world.options.maximum_campaign_size < 20
```

Comment:
Mini-campaign should exclude Very Hards under `default` option

Reply:
Added it.